### DISCUSSION TITLE
Yi app is unable to display SD card saved clips

### DISCUSSION THREAD

**Discussion by way-lo** [alienatedsec/yi-hack-v5#401] [Q&A] [answered]
I don't know if this is a newer development or something that's been an issue and I've never noticed.

I have 2 cams I've flashed to v5.  One is 27US, the other is 48US.

Both flash fine and I can access the web UI and RTSP feeds no problem.

However, when accessing either cam with the original Yi Cam app, I am unable to playback any recorded clips from the SD card.  It could be the continuously recorded feed, or just motion activated clips.  Neither will play.  I drag the timeline back to a certain time, and the playback immediately reverts back to LIVE.  This is on both cameras, so I do not think it is a unique problem with just one cam.

The recording is definitely there, it just cannot be played back within the app.

Am I missing a setting?

**Comment by alienatedsec** [comment, marked as answer]
Please check #385

**Reply by way-lo** [reply]
Thank you, that seems to have fixed it!  Even though there were many recordings, the app just couldn't read them.

Reformatting the SD card, and adding the yi-hack v5 folder back afterwards, and now we're rolling.

As I now recall, I was swapping the same SD card between different cameras, forgetting that the yi-hack folder was not only needed but probably had specific settings for an individual camera.
